src/data/real/pi.lean

```diff
@@ -142,4 +142,122 @@ lemma pi_lt_3141593 : pi < 3.141593 := by pi_upper_bound [
   27720/19601, 56935/30813, 49359/25163, 258754/130003, 113599/56868, 1101994/551163,
   8671537/4336095, 3877807/1938940, 52483813/26242030, 56946167/28473117, 23798415/11899211]
 
+
+noncomputable theory
+open real filter
+open_locale classical big_operators topological_space
+local notation `|`x`|` := abs x
+local notation `π` := real.pi
+
+-- The following theorem will be removed:
+theorem mvt (f f':ℝ→ℝ) {a b c:ℝ} (h : a ≤ b) (hderiv : ∀ (x ∈ set.Icc a b), has_deriv_at f (f' x) x)
+  (hbound : ∀ (x ∈ set.Icc a b), |f' x| ≤ c) : |f b - f a| ≤ c * (b - a) :=
+begin
+  rw ← norm_eq_abs (f b - f a),
+  apply norm_image_sub_le_of_norm_deriv_le_segment',
+  { intros x hx,
+    exact (hderiv x hx).has_deriv_within_at },
+  { intros x hx,
+    rw norm_eq_abs (f' x),
+    exact hbound x (set.Ico_subset_Icc_self hx) },
+  { exact set.right_mem_Icc.mpr h },
+end
+
+
+/-! ### Leibniz's Series for Pi
+`∑ (-1)^i / (2*i+1)` from `i=1` to `n` tends to `π/4` as `n` tends to `+∞` -/
+
+lemma leibniz : tendsto (λ k, ∑ i in finset.range k, ((-(1:ℝ))^i / (2*i+1))) at_top (𝓝 (π/4)) :=
```

Comment:
@digama0 How should we name this result?
Also, should this be formulated in terms of `tsum`?

Reply:
It can't be stated with `tsum` because the series is not absolutely convergent (the precise summation order is important here).

Reply:
Ooh, that's right. It's been a while (-;

Reply:
This could use a different name. `tendsto_sum_pi_div_4`? Not very specific, but maybe good enough.